ui-react/src/utils/zIndexesManager.ts

```diff
@@ -0,0 +1,17 @@
+
+export let zIndexes:any;
```

Comment:
```suggestion
export let zIndexes:Record<ZIndexLayers, number>
```
This way, anyone who will use this object will have amazing DX with auto completion.